src/Adapters/Redis/RedisStorageInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Lamoda\Metric\Adapters\Redis;
+
+interface RedisStorageInterface
+{
+    public function setMetricValue(string $name, float $value, array $tags = []): void;
```

Comment:
for me both these signatures look like `(MetricInterface $metric, float $value): void`. They are only called from places where metric instance exist.

Reply:
`RedisStorageInterface` is dropped in favor of using `Connection` in metric wrapper.